lib/java/android/support/design/internal/NavigationMenuPresenter.java

```diff
@@ -323,13 +323,16 @@ public void onClick(View v) {
           setUpdateSuspended(true);
           MenuItemImpl item = itemView.getItemData();
           boolean result = menu.performItemAction(item, NavigationMenuPresenter.this, 0);
+          boolean checkStateChanged = false;
           if (item != null && item.isCheckable() && result) {
             adapter.setCheckedItem(item);
+            checkStateChanged = true;
           }
           setUpdateSuspended(false);
-          updateMenuView(false);
+          if (checkStateChanged) {
+            updateMenuView(false);
+          }
         }
-      };
```

Comment:
Revert

Reply:
Well. I feel silly.

Thanks for spotting that.